disnake/__init__.py

```diff
@@ -20,7 +20,7 @@
 import logging
 from typing import Literal, NamedTuple
 
-from . import abc, opus, ui, utils
+from . import abc as abc, opus as opus, ui as ui, utils as utils
```

Comment:
I think it's worth adding a comment here

Reply:
done 👍  b4fb8dcedc367a7647ac97fa0b69baf45428f251